EXPLAINER.md

```diff
@@ -74,6 +76,13 @@ But even though we'd like to design the API to eventually enable all these use
 cases, initially we'd almost certainly be shipping a very limited API surface
 with limited capabilities.
 
+We also want to make it possible for websites to get access to some directory
+without having to first prompt the user for access. This helps to make
```

Comment:
I'd order these examples by "hierarchy of constituencies", so put the "write automated tests" example last.

Do we also want to call out the generic use case of storing internal application state in mutable file-like data where neither Indexed DB nor Cache Storage are good fits?

Reply:
swapped the order. Not sure about such generic use cases... Some more specific use case might make sense for that, but I'm not sure if just describing the capability makes for a very compelling use case.